curator/Dockerfile

```diff
@@ -17,22 +26,17 @@ ENV HOME=/opt/app-root/src \
     CURATOR_TIMEOUT=300
 
 ARG LOCAL_REPO
-ADD sources /tmp/lib/
-WORKDIR /tmp/lib
-RUN while read -r hash file ; do \
-      curl -s -o $file http://pkgs.devel.redhat.com/repo/containers/logging-curator5-container/$file/$hash/$file ; \
-    done <sources
-
 RUN if [ -n "${LOCAL_REPO}" ] ; then \
      curl -s -o /etc/yum.repos.d/local.repo ${LOCAL_REPO} ; \
     fi
 
-RUN yum-config-manager --enable rhel-7-server-ose-3.8-rpms \
-                       --enable rhel-7-server-extras-rpms && \
-    INSTALL_PKGS="elasticsearch-curator-5.2.0-1.x86_64.rpm \
-                  python2-ruamel-yaml" && \
+ADD elasticsearch-curator-5.2.0-1.x86_64.rpm /tmp/ocp/
+
+RUN yum install -y --setopt=tsflags=nodocs /tmp/ocp/elasticsearch-curator-5.2.0-1.x86_64.rpm && \
+    INSTALL_PKGS="python2-ruamel-yaml" && \
```

Comment:
I think you still need the
```
RUN yum-config-manager --enable rhel-7-server-ose-3.10-rpms \
--enable rhel-7-server-extras-rpms
```
in order to install python2-ruamel-yaml

Reply:
Per info from @adammhaile, OSBS ignores those repo enablements and provides its own during the build process.  They dont provide value unless building outside of the build system.  The LOCAL_REPO allowed me to build locally by providing my own repo file.   I was able to make a scratch build of curator using this dockerfile without that statement.